packages/mobile/src/harmony-native/components/Button/Button/Button.tsx

```diff
@@ -0,0 +1,266 @@
+import type { ReactNativeStyle } from '@emotion/native'
+import type { TextStyle } from 'react-native/types'
+
+import type { IconProps } from 'app/harmony-native/icons'
+
+import { useTheme } from '../../../foundations/theme'
+import { BaseButton } from '../BaseButton/BaseButton'
+import type { ButtonProps } from '../types'
+import { ButtonSize, ButtonType } from '../types'
+
+const clamp = (val: number, min: number, max: number) => {
+  return Math.max(min, Math.min(val, max))
+}
+
+const adjustHexColor = (color: string, percent: number) => {
```

Comment:
Let's use https://www.npmjs.com/package/color like we currently do in mobile button. Could you pull in the types for it while you're at it?